components/popover_list_members/popover_list_members.jsx

```diff
@@ -80,26 +75,15 @@ export default class PopoverListMembers extends React.Component {
     showMembersModal = (e) => {
         e.preventDefault();
 
-        this.setState({
-            showPopover: false,
-            showChannelMembersModal: true,
-        });
-    };
-
-    hideChannelMembersModal = () => {
-        this.setState({showChannelMembersModal: false});
-    };
-
-    showChannelInviteModal = () => {
-        this.setState({showChannelInviteModal: true});
-    };
+        this.closePopover();
 
-    hideChannelInviteModal = () => {
-        this.setState({showChannelInviteModal: false});
-    };
+        const modalData = {
+            modalId: ModalIdentifiers.CHANNEL_MEMBERS,
```

Comment:
How are we handling the other removed modals?

Reply:
Is this in reference to the methods I deleted (`hideTeamMembersModal`, `showChannelInviteModal`, etc.)? I was very confused as to why these modals were referenced here. I could not find any situation where the Team Members Modal and Channel Invite Modal would ever be active/open when the profile pop-over is used. I tested every situation I could think of and couldn't find a reason to not delete them. I definitely could be missing something though. I asked twice in the Developers channel to be sure but haven't gotten an answer yet.

Reply:
Makes sense, I am also new and not that familiar with the codebase, I would wait for an answer on that, otherwise LGTM 👍

Reply:
I did some digging and I found out why that unnecessary code had existed. At one point, both the header, the mobile header, and the dropdown all had their own versions of those modals. In https://github.com/mattermost/mattermost-webapp/pull/1666, the ChannelHeader and Navbar (the mobile header) components were refactored to remove a bunch of duplication there, but this component was forgotten about so it still used the old way of opening those modals.

Thanks for helping us clean up a bunch of old, dead code!

Reply:
Perfect! Thanks for looking into that.